flaskapi/demo/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# context should be set to dir above demo dir
+FROM python:3.9-alpine
```

Comment:
I wonder whether Dockerfiles should explicitly define the environment variables they need to work properly? In this case, we need four POSTGRES_* env variables, but that's not clear from the Dockerfile.

Reply:
now outdated as demo no longer here